apiary.apib

```diff
@@ -375,84 +379,139 @@ NOT YET !!!!!!
 </pre>
 
 + Request (application/json)
-
     + Headers
 
             API-version: 0.1
             Authorization: Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR - Authentication Bearer Token
 
     + Attributes
-
         + event: association - Socket Event Name
 
 + Response 200 (application/json)
-
     + Attributes (array, fixed-type)
         + (object)
             + seller: `e900b7b0-c097-4ce8-9426-7399d6412b26` - Seller's UUID
             + buyer: `9eef88ba-abf3-4cdc-a2cc-c63e331c4c47` - Buyer's UUID
-            + value: 10 - win value
-            + achievement: 4
+            + value: `10` (number) - win value
+            + achievement: `4` (number)
 
 ## Data Structures
 ### BID (object)
 + id: `b345273d-ffc9-4916-bf7e-5615f0ee8a39` - Message's UUID
-+ start_time: 2019/09/01 13- Bidding Datetime Start
-+ end_time: 2019/09/01 14 - Bidding Datetime End
-+ bid_type: sell - sell or buy
-+ status: 結算完成 - Bidding Status
-+ transaction_hash: 0xdb13d6c572a482f2a9578bfb3cc41a2f0e349852138e788db795ee51ce40b439 - Transaction Hash
-+ upload: 2019/09/01 12:55 - Upload Time
++ date: `2019/09/01` - Trade Target Date
++ time: `13:00-14:00` - Trade Period
++ bid_type: `sell` - sell or buy
++ bids
+    + price: `5.0` (number) - Bidding Price
+    + value: `20.0` (number) - Bidding Value
++ status: `已結算` - Bidding Status
++ transaction_hash: `0xdb13d6c572a482f2a9578bfb3cc41a2f0e349852138e788db795ee51ce40b439` - Transaction Hash
 + counterpart
-    + name: 國立成功大學
-    + address: 701台南市東區大學路1號
-+ average_price: 5 (number)- average unit price
-+ bid_value: 20 (number)- Bidding Value
-+ win
-    + value: 10 (number)- Winning Value
-    + total_price: 50 (number)- Winning Total Price
-+ achievement: 100% - 0~100%
-+ settlement: 50 (number)- Settlement Transaction Price
+    + name: 沙崙綠能科學城C區BEMS
+    + address: 台南市歸仁區高發二路360號C區
++ win: `1` (number) - Deal or No Deal
++ wins
+    + value: `10` (number) - Winning Value
+    + price: `50` (number) - Winning Total Price
++ upload: `Thu, 12 Mar 2020 14:15:00 GMT` - Upload Time
++ achievement: `100%` - 0~100%
++ settlement: `50` (number) - Settlement Transaction Price
+
+### BidSubmit1
++ bid_type: `buy` (string) - 投標類型
++ date: `2020/10/29` (string) - 目標執行日期 YYYY/MM/DD
++ end_time: `2020/10/29 19` (string) - 執行結束時間
++ id：`049a9c28-c0cc-418e-9f35-23f6bc43df7e` - 投標 UUID
```

Comment:
Watch out! Should use : instead of ： !!！!!

Reply:
Noted.
Will be replace by 'find and replace'

Reply:
add a space between `:` and `